testing/time_test.ts

```diff
@@ -70,19 +70,33 @@ Deno.test("Fake Date instance methods passthrough to real Date instance methods"
     const now = new Date("2020-05-25T05:00:00.12345Z");
     assertEquals(now.toISOString(), "2020-05-25T05:00:00.123Z");
 
-    const func = spy(
+    const func1 = spy(
       _internals.Date.prototype,
       "toISOString",
     );
     try {
       now.toISOString();
-      assertSpyCall(func, 0, {
+      assertSpyCall(func1, 0, {
         args: [],
         returned: "2020-05-25T05:00:00.123Z",
       });
-      assertInstanceOf(func.calls[0].self, _internals.Date);
+      assertInstanceOf(func1.calls[0].self, _internals.Date);
     } finally {
-      func.restore();
+      func1.restore();
+    }
+
+    const func2 = spy(
+      _internals.Date.prototype,
+      Symbol.toPrimitive,
+    );
+    try {
+      `${now}`;
+      assertSpyCall(func2, 0, {
+        args: ["string"],
+      });
+      assert(typeof func2.calls[0].returned === "string");
```

Comment:
maybe it would be better to do 0+now, I think that would cause Symbol.toPrimitive to be called with "number" and return a number value which would be consistent across systems.

Reply:
This seems to come with a few other problems. First I'd have to cast `now as unknown as number` in order to not get a type error. And it seems to call toPrimitive with `"default"` rather than `"number"`, so it ends up returning the same string as with the `${now}` approach.

But I looked at the spec for a bit and it seems Date.toPrimitive is pretty well defined, so I think it'll be pretty safe to test for the exact string.

Reply:
Ah never mind, seems like `Number(now)` calls it with argument `"number"`